packages/flutter/lib/src/widgets/page_storage.dart

```diff
@@ -130,8 +130,12 @@ class PageStorageBucket {
 /// Usually you don't need to explicitly use a [PageStorage], since it's already
 /// included in routes.
 ///
-/// [PageStorageKey] is used by [Scrollable] if
-/// `keepScrollOffset` is enabled to save their [ScrollPosition]s.
+/// [PageStorageKey] is used by [Scrollable] if [ScrollController.keepScrollOffset]
+/// is enabled to save their [ScrollPosition]s. When more than one
+/// scrollable ([ListView], [SingleChildScrollView], [TextField], etc.) appears
+/// in the same route, if you want to save all of their positions,
```

Comment:
Are you sure if it's only when they appear in the same route, or should it be global unique?

Reply:
Not need global unique, but should equal during rebuilding.

Reply:
If you like you can give them the same key, it works well. Because the identifier is a key chain

Reply:
Hi, I update the issue Description above.

Reply:
Actually, "in the same route" is not precise enough. 
```suggestion
/// within the widget's closest ancestor [PageStorage] (such as within the same route), if you want to save all of their positions,
```

Reply:
Actually, "in the same route" is not precise enough, isn't it?
```suggestion
/// within the widget's closest ancestor [PageStorage] (such as within the same route), if you want to save all of their positions,
```

Reply:
Also "if you want to save all of their positions" should be "if you want to save all of their positions independently" followed by "unique [PageStorageKey]s". Otherwise, their positions ARE stored, but overridden by each other.

Reply:
NICE suggesting.

Reply:
Done.